### ISSUE TITLE
Any further progress?

### ISSUE THREAD

**Issue by EricE** [madenwala/Moen.U.Api#1]
Have you been able to make any more progress on the interface?  Looking at possibly designing a Moen U into a new house and want to control it from Home Assistant.

**Comment by tuday2** [comment]
I've been also looking at this to control my existing MoenU. I was trying to figure out how to turn this into an AddOn for Home Assistant. Have you made any further progress?

**Comment by lockyt** [comment]
Yes I would be interested too.  Thinking of buying on of these for my HA home..

**Comment by yungeconomist** [comment]
This guy was able to do make a plugin that shuts on and off a Flo by Moen. I wonder if there are some takeaways what might be helpful here. 

I'll be looking through both to see if I can find some commonality.
 https://www.reddit.com/r/homebridge/comments/hpkijk/using_flo_by_moen_with_homebridge/

**Comment by EricE** [comment]
Partial interface: https://thesmartestcondo.com/11-review-smart-shower-u-by-moen/

**Comment by ZinkNotTheMetal** [comment]
Yeah I wrote that blog
